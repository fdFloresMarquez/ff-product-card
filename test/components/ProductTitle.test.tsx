import React from 'react';
import renderer from 'react-test-renderer';

import ProductCard, { ProductTitle } from '../../src/components';
import { product } from '../data/products';

describe('Product Title', () => {
  test('should render without crashing with the Product Name', () => {
    const wrapper = renderer.create(
      <ProductCard product={product}>{() => <ProductTitle />}</ProductCard>
    );

    expect( wrapper.toJSON() ).toMatchSnapshot();
  });
});
