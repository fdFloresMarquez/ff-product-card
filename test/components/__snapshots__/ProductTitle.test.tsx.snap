// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Title should render without crashing with the Product Name 1`] = `
<div
  className="productCard undefined"
>
  <span
    className="productDescription undefined"
  >
    Coffee Mug - Card
  </span>
</div>
`;
